model/task_lifecycle.go

```diff
@@ -43,6 +43,9 @@ func SetActiveState(t *task.Task, caller string, active bool) error {
 	}
 
 	if active {
+		if t.Requester == evergreen.MergeTestRequester {
```

Comment:
Should we case on if caller == mci? SetActiveState is sometimes used internally to activate dependencies; since it's a commit queue task it theoretically would already have any dependencies activated but it might be safe to check, since caller == mci isn't manually scheduled.

Reply:
Yep, though since the caller is set to a variety of different things I don't think checking that is foolproof. I moved it to the UI entry points instead